vocabularies/qld-crs.ttl

```diff
@@ -9,90 +9,89 @@
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 <http://linked.data.gov.au/def/queensland-crs> a owl:Ontology , skos:ConceptScheme ;
-    dct:created "2020-01-30T15:57:41"^^xsd:dateTime ;
+    dct:created "2020-03-13T15:57:41"^^xsd:dateTime ;
     dct:creator "Geological Survey of Queensland" ;
-    dct:modified "2020-01-30T15:57:41"^^xsd:dateTime ;
+    dct:modified "2020-03-13T15:57:41"^^xsd:dateTime ;
     dct:source "Compiled by the Geological Survey of Queensland" ;
     skos:definition "The coordinate reference systems used and refered to within Queensland"@en ;
-    skos:hasTopConcept qldcrs:australian-geodetic-datum-1966,
-        qldcrs:australian-geodetic-datum-1984,
-        qldcrs:australian-map-grid-1966,
-        qldcrs:australian-map-grid-1984,
-        qldcrs:geocentric-datum-of-australia-1994,
-        qldcrs:geocentric-datum-of-australia-2020,
-        qldcrs:map-grid-of-australia-1994,
-        qldcrs:map-grid-of-australia-2020 ;
+    skos:hasTopConcept qldcrs:AGD66,
+        qldcrs:AGD84,
+        qldcrs:AMG66,
+        qldcrs:AMG84,
+        qldcrs:GDA94,
+        qldcrs:GDA2020,
+        qldcrs:MGA94,
+        qldcrs:MGA2020 ;
     skos:prefLabel "Queensland CRS"@en .
 
 qldcrs:geographic a skos:Collection ;
     skos:definition "A geographic coordinate system (GCS) is a coordinate system which uses a three-dimensional spherical surface (ellipsoid) to define locations on the earth. A common choice of coordinates is latitude and longitude."@en ;
-    skos:member qldcrs:australian-geodetic-datum-1966,
-        qldcrs:australian-geodetic-datum-1984,
-        qldcrs:geocentric-datum-of-australia-1994,
-        qldcrs:geocentric-datum-of-australia-22 ;
+    skos:member qldcrs:AGD66,
+        qldcrs:AGD84,
+        qldcrs:GDA94,
+        qldcrs:GDA2020 ;
     skos:prefLabel "Geographic"@en .
 
 qldcrs:projected a skos:Collection ;

```

Comment:
Should be named "projected coordinate systems"
has prefLabel: "Projected Coordinate System" @en
has alfLabel: "Projected Coordinate Reference System" @en ; "Projected CRS" @en